main/tzutils/template.py

```diff
@@ -1,18 +1,15 @@
 pkgname = "tzutils"
 pkgver = "2024a"
-pkgrel = 3
+pkgrel = 4
 build_style = "makefile"
 make_build_args = ["KSHELL=/bin/sh"]
 make_install_args = ["ZICDIR=/usr/bin", "ZFLAGS=-b fat"]
-hostmakedepends = []
```

Comment:
is wrong

Reply:
ah right, missed that it's modified below